### PR TITLE
Add Team Members to README.md

### PR DESCRIPTION
**Pull request by tommyrharper**
The idea behind this ticket is that we should credit each team member on the repository.

It simply adds a list of team members to the README.md

### CODE REVIEW

**Comment by paulochoi** [comment]
Pull request approved.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # ultimate-logger
+
 View your logs from the Front-End and Back-End in one place.
+
+## Team Members
```
Comment (tommyrharper) [inline README.md:5]: Nice title choice Tom.

**Review by paulochoi — APPROVED** [review]
Approved

**Review by ayemmoe — APPROVED** [review]

**Review by ayemmoe — APPROVED** [review]

**Review by paulochoi — APPROVED** [review]
Re-approving.

**Review by ayemmoe — APPROVED** [review]

**Review by paulochoi — APPROVED** [review]
Re-approving(2)

**Pull request merged**